### ISSUE TITLE
abyss 2.1.4 fails to build on macOS with GCC 8

### ISSUE THREAD

**Issue by sjackman** [bcgsc/abyss#266]
```
g++-8 -Wall -Wextra -Werror -g -O2  -L/usr/local/opt/open-mpi/lib  -L/usr/local/opt/google-sparsehash/lib  -o abyss-paired-dbg-mpi abyss_paired_dbg_mpi-parallelAbyss.o abyss_paired_dbg_mpi-CommLayer.o abyss_paired_dbg_mpi-NetworkSequenceCollection.o abyss_paired_dbg_mpi-MessageBuffer.o abyss_paired_dbg_mpi-Messages.o ../Assembly/libassembly.a ../Common/libcommon.a ../DataLayer/libdatalayer.a ../DataBase/libdb.a -lsqlite3 -ldl -lm  -lmpi  -ldl -lm 
make[2]: Nothing to be done for `install-data-am'.
 .././install-sh -c -d '/usr/local/Cellar/abyss/2.1.4/bin'
  /usr/bin/install -c ABYSS-P abyss-paired-dbg-mpi '/usr/local/Cellar/abyss/2.1.4/bin'
Making install in Bloom
g++-8 -DHAVE_CONFIG_H -I. -I..  -I.. -I../Common -I../DataLayer -I/private/tmp/abyss-20181112-25614-1pqmx43/abyss-2.1.4 -isystem/usr/local/opt/boost/include -isystem/usr/local/opt/boost/include/include -isystem/usr/local/opt/open-mpi/include  -isystem/usr/local/opt/google-sparsehash/include  -isystem/private/tmp/abyss-20181112-25614-1pqmx43/abyss-2.1.4/boost_1_56_0 -Wall -Wextra -Werror -fopenmp -g -O2 -c -o abyss_bloom-bloom.o `test -f 'bloom.cc' || echo './'`bloom.cc
In file included from bloom.cc:24:
../BloomDBG/HashAgnosticCascadingBloom.h: In member function 'bool HashAgnosticCascadingBloom::contains(const std::vector<long unsigned int>&) const':
../BloomDBG/HashAgnosticCascadingBloom.h:103:40: error: no matching function for call to 'BloomFilter::contains(const std::vector<long unsigned int>&)'
   return m_data.back()->contains(hashes);
                                        ^
```
See https://gist.github.com/sjackman/7eb70b19267515448d67f6aa6e986d54#file-02-make-L168-L178

This error is after cherry-picking this commit https://github.com/bcgsc/abyss/commit/195f19bba03fec18d86cd931b34275222ba667fc

Is CircleCI currently passing? Let's get it passing again if not.

**Comment by benvvalk** [comment]
@sjackman Also, understood about getting CircleCI back into shape. I intended to fix it before I left anyway.

**Comment by sjackman** [comment]
```sh
./configure CC=/Users/sjackman/.homebrew/bin/gcc-8 CXX=/Users/sjackman/.homebrew/bin/g++-8
```

**Comment by benvvalk** [comment]
Great. Thanks, @sjackman!

If that doesn't work, I'll try using the group laptop.

**Comment by benvvalk** [comment]
fixed by 97d901810757137b9321a595d3bbfa38cd5e2e68

Thanks for your kindness and patience, @sjackman!

**Comment by sjackman** [comment]
Thanks, Ben!
